cypress/integration/org_registration.spec.js

```diff
@@ -87,6 +88,8 @@ context("org registration", () => {
     });
 
     it("prevents the user from registering an unavailable org name", () => {
+      cy.registerOrg("coolname");
```

Comment:
It would be nice if this would be decoupled from the other test.

Reply:
@rudolfs Could you expand what you mean exactly?

Reply:
Ah, never mind, I missed the `nukeRegistryState` in the before. All good!